src/structures/paging/mapper/offset_page_table.rs

```diff
@@ -39,6 +41,7 @@ struct PhysOffset {
 }
 
 impl PhysToVirt for PhysOffset {
+    #[inline]
```

Comment:
These `#[inline]` attributes are fine with me, but perhaps mention their addition in the pull request/commit description.